credentials/src/main/kotlin/web5/sdk/credentials/util/JwtUtil.kt

```diff
@@ -0,0 +1,131 @@
+package web5.sdk.credentials.util
+
+import com.nimbusds.jose.JOSEObjectType
+import com.nimbusds.jose.JWSAlgorithm
+import com.nimbusds.jose.JWSHeader
+import com.nimbusds.jose.jwk.JWK
+import com.nimbusds.jose.util.Base64URL
+import com.nimbusds.jwt.JWTClaimsSet
+import com.nimbusds.jwt.JWTParser
+import com.nimbusds.jwt.SignedJWT
+import foundation.identity.did.DIDURL
+import web5.sdk.common.Convert
+import web5.sdk.crypto.Crypto
+import web5.sdk.dids.Did
+import web5.sdk.dids.DidResolvers
+import web5.sdk.dids.findAssertionMethodById
+import java.security.SignatureException
+
+private const val JsonWebKey2020 = "JsonWebKey2020"
```

Comment:
this needs to be updated to `JsonWebKey` https://github.com/TBD54566975/sdk-development/issues/73

Reply:
yes I'll make another PR that does this, since we will need to update lots of test vectors too